test/emqx_protocol_v5_SUITE.erl

```diff
@@ -14,7 +14,7 @@
 %% limitations under the License.
 %%--------------------------------------------------------------------
 
--module(mqtt_protocol_v5_SUITE).
+-module(emqx_protocol_v5_SUITE).
```

Comment:
nit, I would prefer `emqx_mqtt_protocol_v5`, I just feel that term `mqtt_protocol_v5` is unbreakable.